webapi/src/test/scala/org/knora/webapi/slice/admin/domain/service/ProjectADMServiceSpec.scala

```diff
@@ -46,7 +47,8 @@ object ProjectADMServiceSpec extends ZIOSpecDefault {
           shortname = shortname,
           shortcode = shortcode,
           longname = None,
-          description = List(StringLiteralV2("description not used in test but is required by constructor", None)),
+          description =
+            NonEmptyChunk(StringLiteralV2("description not used in test but is required by constructor", None)),
```

Comment:
Just curious, why do you prefer `NonEmptyChunk` over `List`?

Reply:
Ah, only now I've read the description of this PR: because it's not allowed to be empty, right?

Reply:
Correct.